OpenPods/app/src/main/java/com/dosse/airpods/PodsService.java

```diff
@@ -93,6 +93,7 @@ private void startAirPodsScanner () {
                 Log.d(TAG, "START SCANNER");
 
             SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
+            isWidgetActive = prefs.getBoolean("isWidgetActive", false); // Check if the widget is active in case the app is stopped
```

Comment:
Wouldn't it make more sense to use a static variable instead of sharedprefs? I mean, the fact that the widget is active isn't a preference.

Reply:
If the app is stopped, the variable would be reset until the widget is readded to the home screen. The onEnabled function in the AppWidgetProvider is only called when the widget is added, not when the app is reopened (even in the background)
https://developer.android.com/guide/topics/appwidgets#AppWidgetProvider